dockorm/container.py

```diff
@@ -103,8 +103,9 @@ def format_links(self):
         return {}
 
     volumes_readwrite = Dict()
-
     volumes_readonly = Dict()
+    volumes_no_bind = []
```

Comment:
Should we use traitlets for these, so they can be configured with all the other settings?